openlibrary/macros/StarRatingsStats.html

```diff
@@ -6,7 +6,11 @@
 <ul class="readers-stats" itemprop="aggregateRating" itemscope itemtype="https://schema.org/AggregateRating">
   <li class="avg-ratings">
     $if rating_stats.get('avg_rating'):
-      $:('<span class="readers-stats__star">★</span>' * int(rating_stats.get('avg_rating'))) <span itemprop="ratingValue">$(rating_stats['avg_rating'])</span>
+      $ stats_decimal = (float(rating_stats.get('avg_rating')))-(int(rating_stats.get('avg_rating')))
+      $:('<span class="readers-stats__star">★</span>' * int(rating_stats.get('avg_rating')))
+      $if (stats_decimal >= 0.5) and (stats_decimal < 1):
+        $:('<span class="readers-stats__star--half">★</span>')
```

Comment:
Maybe in future we should also output the empty stars (but out of scope for now :))